lib/migrate.test.js

```diff
@@ -23,13 +23,14 @@ describe('migrate', () => {
       throw new Error()
     })
 
-    const mockExit = jest.spyOn(process, 'exit').mockImplementation(() => {})
     const mockError = jest.spyOn(console, 'error').mockImplementation(() => {})
 
     Postgrator.mockImplementation(() => ({ migrate: mockMigrate }))
 
     await migrateSchema()
 
+    expect(process.exitCode).toEqual(1)
+
     expect(mockMigrate).toHaveBeenCalledTimes(1)
     expect(mockError).toHaveBeenCalledTimes(1)
     expect(mockExit).toHaveBeenCalledWith(1)
```

Comment:
This line needs to be removed as `mockExit` was removed and the `system.exit(..)` method is no longer called.

Reply:
Good catch :-)

Reply:
Done!